src/scheduling.py

```diff
@@ -40,16 +40,17 @@ def logout():
         response.set_cookie(USER_ID_COOKIE, "", expires=0)
         return response
 
-    @app.route("/login", methods=["POST"])
+    @app.route("/login", methods=["POST"])  # using POST method for user login
     def login():
-        email = request.form["email"]
+        email = request.form["email"]  # Request module is used to fetch users "email" and "password"
```

Comment:
We've also discovered that our PR lint checks are not failing. Please run. Black version 21.12b0 on the source before submitting and then run pylint version 2.12.2 and flake8 version 4.0.1 to verify it is clean. 

On a macOS system, you should be able to run `./pr_build.sh fix` to do all these actions.

file would be reformatted, 1 file would be left unchanged.

************* Module scheduling
src/scheduling.py:45:0: C0301: Line too long (101/100) (line-too-long)
src/scheduling.py:48:0: C0301: Line too long (113/100) (line-too-long)
src/scheduling.py:83:0: C0301: Line too long (115/100) (line-too-long)
src/scheduling.py:101:0: C0303: Trailing whitespace (trailing-whitespace)
src/scheduling.py:93:8: W0612: Unused variable 'user' (unused-variable)
src/scheduling.py:172:4: E0102: function already defined line 92 (function-redefined)

Reply:
git checkout my_name/meaningful_change_name
can you help me in this.
I am little bit confused how to do it.

Reply:
What is to be written in "my_name"

Reply:
what is to be written in "meaningful_change_name"